packages/ckeditor5-language/src/textpartlanguageui.ts

```diff
@@ -106,6 +106,16 @@ export default class TextPartLanguageUI extends Plugin {
 				return ( value && titles[ value ] ) || defaultTitle;
 			} );
 
+			dropdownView.buttonView.bind( 'ariaLabel' ).to( languageCommand, 'value', value => {
+				const selectedLanguageTitle = value && titles[ value ];
+
+				if ( !selectedLanguageTitle ) {
+					return accessibleLabel;
+				}
+
+				return `${ accessibleLabel }, ${ selectedLanguageTitle }`;
```

Comment:
In the heading feature you went with "Current value, Feature name" ("Heading 2, Heading") (as per [Tomek's comment](https://github.com/cksource/ckeditor5-commercial/issues/6029)). Here, you have the opposite: "Feature name, Current value" ("Language, French"). I think text part language should be aligned to headings in this matter.

Reply:
Agh, you are correct. I'm fixing it.

Reply:
Fixed.